oak/server/rust/oak_runtime/src/node/external.rs

```diff
@@ -23,10 +23,15 @@ use std::{sync::RwLock, thread};
 // mechanism when the runtime's main() is in Rust.
 pub type NodeFactory = fn(config_name: &str, node_id: NodeId, handle: oak_abi::Handle);
 
+// Global callback used to create pseudo-Nodes of types that are not
+// supported by the Rust Runtime.
```

Comment:
I think this doc should be on `FACTORY`, not on the `lasy_static` block, or is it not allowed inside the macro?